pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -92,12 +92,16 @@ select.pf-c-form-control {
   // https://github.com/patternfly/patternfly/issues/3959
   --pf-c-card__header-toggle--MarginTop: 0;
 
-  .pf-c-card__header:not(.ct-card-expandable-header) {
+  .pf-c-card__header:not(.ct-card-expandable-header),
+  .pf-c-card__header:not(.ct-card-expandable-header) .pf-c-card__header-main {
     // upstream fix (pending): https://github.com/patternfly/patternfly/pull/3714
+    display: flex;
     flex-wrap: wrap;
     row-gap: var(--pf-global--spacer--sm);
     justify-content: space-between;
+  }
 
+  .pf-c-card__header:not(.ct-card-expandable-header) {
```

Comment:
This should be a separate commit with a different justification, as it's not (obviously) related to "introduce PF5 overrides"